fil_fungible_token/src/token/mod.rs

```diff
@@ -415,24 +423,36 @@ where
         // spender must be an id address
         let spender = expect_id(spender)?;
         // resolve owner and receiver
-        let owner = self.resolve_to_id(owner)?;
-        let receiver = self.resolve_to_id(receiver)?;
+        let owner_id = self.resolve_to_id(owner)?;
+        let receiver_id = self.resolve_to_id(receiver)?;
```

Comment:
Nit: just because "receiver" is overloaded by the syscalls to refer to the message receiver (in the OO sense, i.e. currently executing actor), let's use a different word here. Either `from`/`to` consistently is probably best, even if awkward English. Can defer to follow-up